code/src/pages/StartScreen.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import maze from '../reducers/maze'
+import { firstFetch } from '../reducers/maze'
+
+const StartScreen = () => {
+  const [inputValue, setInputValue] = useState('')
+
+  const dispatch = useDispatch();
+
+  const isLoading = useSelector(store => store.maze.isLoading)
+
+  const onStartGame = () => {
+      dispatch(maze.actions.setLoading(!isLoading))
+      dispatch(maze.actions.setUsername(inputValue))
+      dispatch(firstFetch());
+  }
+  
+  return (
+    <Main>
+      <Border>
+        <Greeting>
+          Welcome traveller!
+        </Greeting>
+        <Instruction>
+          Enter your name and start the adventure.
+        </Instruction>
+        <Label
+          htmlFor='name_input'>
+        <NameInput
+          type='text'
+          id='name_input'
+          value={inputValue}
+          onChange={event => setInputValue(event.target.value)}
+        />
```

Comment:
It is not nice that you forgot about the form tag. It striped your input from submission on enter, what is the most used sending the form method 😭